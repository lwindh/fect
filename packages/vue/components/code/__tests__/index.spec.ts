import Code from '../index'
import { mount } from '@vue/test-utils'

describe('Code', () => {
  it('should be render as element', () => {
    const wrapper = mount(Code)
    expect(wrapper.html()).toMatchSnapshot()
    expect(() => wrapper.unmount()).not.toThrow()
  })
  it('should support props block', () => {
    const wrapper = mount(Code, {
      props: {
        block: true,
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.find('pre')).toBeTruthy()
  })
})
