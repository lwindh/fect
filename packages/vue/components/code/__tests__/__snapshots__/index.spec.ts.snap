// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code should be render as element 1`] = `<code><!----></code>`;

exports[`Code should support props block 1`] = `<pre><code><!----></code></pre>`;
